Scripts/build.swift

```diff
@@ -1,178 +0,0 @@
-#!/usr/bin/env swift
-
-import Foundation
-
-// Usage: build.swift <spm|xcode> <platform> [<path_to_xcpretty>]
-
-func execute(commandPath: String, arguments: [String], pipedTo pipeProcess: Process? = nil) throws {
-	let task = Process()
-	task.launchPath = commandPath
-	task.arguments = arguments
-
-	let argumentsString = arguments
-		.map { argument in
-			if argument.contains(" ") {
-				return "\"\(argument)\""
-			} else {
-				return argument
-			}
-		}
-		.joined(separator: " ")
-
-	if let pipeProcess = pipeProcess, let pipePath = pipeProcess.launchPath {
-		let pipe = Pipe()
-		task.standardOutput = pipe
-		pipeProcess.standardInput = pipe
-
-		print("Launching command: \(commandPath) \(argumentsString) | \(pipePath)")
-
-	} else {
-		print("Launching command: \(commandPath) \(argumentsString)")
-	}
-
-	task.launch()
-
-	pipeProcess?.launch()
-
-	task.waitUntilExit()
-
-	guard task.terminationStatus == 0 else {
-		throw TaskError.code(task.terminationStatus)
-	}
-}
-
-enum TaskError: Error {
-	case code(Int32)
-}
-
-enum Platform: String, CustomStringConvertible {
-	case iOS_13
-
-	var destination: String {
-		switch self {
-		case .iOS_13:
-			return "platform=iOS Simulator,OS=13.7,name=iPhone 11 Pro"
-		}
-	}
-
-	var derivedDataPath: String {
-		return ".build/derivedData/\(rawValue)"
-	}
-
-	var description: String {
-		return rawValue
-	}
-}
-
-enum Task: String, CustomStringConvertible {
-	case spm
-	case xcode
-
-	var workspace: String? {
-		switch self {
-		case .xcode:
-			return "Example/Stagehand.xcworkspace"
-		case .spm:
-			return nil
-		}
-	}
-
-	var project: String? {
-		switch self {
-		case .spm:
-			return "generated/Stagehand.xcodeproj"
-		case .xcode:
-			return nil
-		}
-	}
-
-	var scheme: String {
-		switch self {
-		case .xcode:
-			return "Stagehand Demo App"
-		case .spm:
-			return "Stagehand-Package"
-		}
-	}
-
-	var shouldGenerateXcodeProject: Bool {
-		switch self {
-		case .spm:
-			return true
-		case .xcode:
-			return false
-		}
-	}
-
-	var shouldRunTests: Bool {
-		switch self {
-		case .spm:
-			return false
-		case .xcode:
-			return true
```

Comment:
you can see here that we were only running _tests_ on the `Stagehand Demo App`, which is no longer built or tested in CI.